WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostDetailFragment.kt

```diff
@@ -26,6 +26,7 @@ import android.webkit.WebView
 import android.widget.ImageView.ScaleType.CENTER_CROP
 import android.widget.ProgressBar
 import android.widget.TextView
+import android.widget.Toast
```

Comment:
I've noticed that the import triggers a `detekt` `[UnusedImports]` error.

Reply:
Fixed!

Reply:
Thank you for your work on this @07jasjeet 🙇 
I'll proceed with merging once the CI checks complete 🎉